docs/examples/supported_lifecyle_types/1.json

```diff
@@ -0,0 +1,53 @@
+{
+  // Notice: All Key's are case sensitive
```

Comment:
I don't think json supports comments like this (github also highlights it in red)